rpc/src/main/scala/org/bitcoins/rpc/config/BitcoindInstance.scala

```diff
@@ -26,15 +31,123 @@ object BitcoindInstance {
       uri: URI,
       rpcUri: URI,
       authCredentials: BitcoindAuthCredentials,
-      zmqPortOpt: Option[Int])
-      extends BitcoindInstance
+      zmqConfig: ZmqConfig = ZmqConfig()
+  ) extends BitcoindInstance
 
   def apply(
       network: NetworkParameters,
       uri: URI,
       rpcUri: URI,
       authCredentials: BitcoindAuthCredentials,
-      zmqPortOpt: Option[Int] = None): BitcoindInstance = {
-    BitcoindInstanceImpl(network, uri, rpcUri, authCredentials, zmqPortOpt)
+      zmqConfig: ZmqConfig = ZmqConfig()
+  ): BitcoindInstance = {
+    BitcoindInstanceImpl(network,
+                         uri,
+                         rpcUri,
+                         authCredentials,
+                         zmqConfig = zmqConfig)
+  }
+
+  private val DEFAULT_DATADIR =
+    Paths.get(Properties.userHome, ".bitcoin")
+
+  private val DEFAULT_CONF_FILE = DEFAULT_DATADIR resolve "bitcoin.conf"
```

Comment:
Let's explicitly use dot notation as in 

```scala
DEFAULT_DATADIR.resolve("bitcoin.conf")
``` 

infix notation is discouraged as it can be confusing unless you are doing basic arithmetic operations